include/bout/physicsmodel.hxx

```diff
@@ -325,7 +325,7 @@ private:
   }
 
 /// Macro to replace solver->add, passing variable name
-#define SOLVE_FOR(var) solver->add(var, #var)
+#define SOLVE_FOR1(var) solver->add(var, #var);
 #define SOLVE_FOR2(var1, var2) { \
```

Comment:
Do you want to deprecate these specific count versions (for count >1) or do you expect we will keep these?